.github/workflows/tests-evm-rpc.yaml

```diff
@@ -1,20 +1,71 @@
 name: Tests - EVM
+
 on:
   workflow_dispatch:
+  pull_request:
+    branches:
+      - feature/evm
+  push:
+    branches:
+      - feature/evm
+    tags:
+      - evm
+
 env:
-  NODE_URL: "http://35.187.53.161:20551/"
-  AUTH_ID: ${{ secrets.INFURA_GOERLI_PROJECT_ID }}
-  AUTH_PASS: ${{ secrets.INFURA_GOERLI_API_KEY }}
+  NODE_URL: "http://127.0.0.1:19551/"
+
 jobs:
+  build-binaries:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install dependencies
```

Comment:
Can you use the workflow similar to https://github.com/DeFiCh/ain/blob/feature/evm/.github/workflows/build-dev.yaml

```
    - name: Populate environment
      run: GIT_VERSION=1 ./make.sh ci-export-vars

    - name: Build and package
      run: GIT_VERSION=1 TARGET="x86_64-pc-linux-gnu" ./make.sh docker-release

```

This should provide a more coherent build. 
- Removing `GIT_VERSION` can provide the build under a predicatable `./build/latest` if needed